### PR TITLE
hasura-authentication: update Logto integration guide

### PR DESCRIPTION
**Pull request by wangsijie**
### Description

update Logto integration guide

### CODE REVIEW

**Review by charIeszhao — APPROVED** [review]

**Comment by charIeszhao** [comment]
@praveenweb Hi Praveen, we are updating our Logto guide, can you take a look and kindly approve it? Thanks!

File: tutorials/backend/hasura-authentication/tutorial-site/content/integrations/logto.md [hunk 1]
```diff
@@ -10,33 +10,43 @@ Logto is an open source identity solution that helps you build the sign-in exper
...
+
+- A ready-to-use management API can serve as your authentication provider, eliminating the need for extra implementation.
+- SDKs that can integrate your apps with Logto quickly, multi-platform and language compatible, tailored to your development environment.
+- Flexible connectors, scalable with community contributions, customizable with SMAL, OAuth, and OIDC protocols.
+
+💻 **Enterprise-ready solutions**
+
+- RBAC to control your resource through scalable role authorization for diverse use cases.
+- User management and audit Logs to understand identity related user info and keep your security on track.
+- We are currently working on SSO, Organizations and MFA! Stay tuned!
```
Comment (praveenweb) [inline tutorials/backend/hasura-authentication/tutorial-site/content/integrations/logto.md:30]: Can we remove this part? The guide talks from Hasura pov and hence the context here could be misunderstood.

**Review by praveenweb — CHANGES_REQUESTED** [review]
I have added one feedback. We can merge once that's addressed.

**Comment by wangsijie** [comment]
@praveenweb Thanks, I've removed that block, PTAL

File: tutorials/backend/hasura-authentication/tutorial-site/content/integrations/logto.md (hunk 1 above)
Comment (wangsijie) [reply tutorials/backend/hasura-authentication/tutorial-site/content/integrations/logto.md]: updated

**Pull request merged**